bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -435,9 +436,32 @@ func (r *KubeadmConfigReconciler) handleClusterNotInitialized(ctx context.Contex
 		InitConfiguration:    initdata,
 		ClusterConfiguration: clusterdata,
 		Certificates:         certificates,
-	})
+	}
+
+	var cloudInitData []byte
+
+	switch scope.Config.Spec.Format {
+	case bootstrapv1.Ignition:
+		ign, warnings, err := ignition.NewInitControlPlane(&ignition.ControlPlaneInput{
+			ControlPlaneInput: controlPlaneInput,
+			Ignition:          scope.Config.Spec.Ignition,
+		})
+		if err != nil {
+			scope.Error(err, "Failed to generate Ignition user data for bootstrap control plane")
+			return ctrl.Result{}, err
+		}
+
+		if warnings != "" {
+			conditions.MarkFalse(scope.Config, bootstrapv1.DataSecretAvailableCondition, bootstrapv1.IgnitionHasWarningsReason, clusterv1.ConditionSeverityWarning, warnings)
```

Comment:
I have not tested this, but I fear this reason will never surface to the user because we are going to override it when calling `r.storeBootstrapData` before the end of this function.
Am I wrong?

Reply:
You might be right. As discussed in other threads, I'll remove this.

Reply:
Removed, so resolving.